### DISCUSSION TITLE
Control change off

### DISCUSSION THREAD

**Discussion by Bipman1** [mido/mido#466] [Q&A] [answered]
Hi

All going well and a great library!  When using the control change item for example to change the pitch bend wheel :-
`on = Message('control_change',control=1,value=val1)`

I presume I only need to send this via port.send(on) and don't need an 'off' signal or a note length?

Bipman

**Comment by Bipman1** [comment, marked as answer]
Right, got that working  now!

**Reply by rdoursenaud** [reply]
Glad to hear that! Hope you enjoy using mido.

FWIW: Control Changes are instantaneous controls and thus don't have an intrinsic on/off status nor any duration. Of course, the implementer (i.e.: hardware or software synth) can still decide that some specific values represent an internal on/off status. Refer to their manual or implementation chart.
